src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReader.cs

```diff
@@ -154,27 +154,39 @@ Assembly[] LoadConfigurationAssemblies()
                 }
             }
 
-            foreach (var library in _dependencyContext.RuntimeLibraries)
+            foreach (var assemblyName in GetSerilogConfigurationAssemblies())
             {
-                if (library.Name != null && library.Name.ToLowerInvariant().Contains("serilog"))
-                {
-                    var assumedName = new AssemblyName(library.Name);
-                    var assumed = Assembly.Load(assumedName);
-                    if (assumed != null && !assemblies.ContainsKey(assumed.FullName))
-                        assemblies.Add(assumed.FullName, assumed);
-
-                    foreach (var assemblyRef in library.Assemblies)
```

Comment:
Regarding this lines:
1) **Assemblies** collection is always empty (see [this](https://github.com/dotnet/cli/blob/rel/1.0.0/src/Microsoft.Extensions.DependencyModel/RuntimeLibrary.cs#L40-L45))
2) **RuntimeLibraries** has full graph in it (see updated Sample project: it references **Serilog.Sinks.RollingFile**, but configures **Serilog.Sinks.File** sink succesfully)